.github/workflows/build_package.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       CIBW_MANYLINUX_X86_64_IMAGE: stellaraccident/manylinux2014_x86_64-bazel-5.0.0:latest
       # CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
 
-      CIBW_BUILD: "cp37-* cp38-* cp39-*"
+      CIBW_BUILD: "cp37-* cp38-* cp39-* cp310-*"
```

Comment:
found a few other spots that could benefit from updates, though they are separate from the release builds:

https://github.com/google/iree/blob/dd860d35a562f826fdbc8a536e5e64457774efe4/.github/workflows/macos_oneshot_build.yml#L12-L16

https://github.com/google/iree/blob/dd860d35a562f826fdbc8a536e5e64457774efe4/build_tools/python_deploy/manylinux_foreach_py.sh#L19-L22

(just searched the project for "cp39")

Reply:
These are antiquated. I'll delete in a followup.

Reply:
can I fix up the macos build ?